data/hai/hai missions.txt

```diff
@@ -14,35 +14,107 @@ mission "First Contact: Hai"
+				goto questions
+			label peaceful
+			`	"We are a people of peace, and you are free to visit us or even live among us. Our worlds provide enough bounty for all."`
+			`	You glance quickly at the human merchant to see his expression, wondering if these aliens can indeed be so benign, but his expression is calm and untroubled. "It's true," he says, "look around you. These buildings have stood for thousands of years. Hai society is so advanced, they need almost no resources to maintain it."`
+			choice
+				`	"Why do you allow humans to settle here?"`
+				`	"What do you ask for in return for letting humans settle here?"`
+			`	"Humans are a young species," the Hai says, "full of energy, full of new ideas. And the Hai are old, and everything we do is what we have done before. When humans go on vacation, they travel to a world with perfect weather, sunny every day. When Hai go on vacation, we visit the stormiest world, to find unpredictability and change. Humans are so strange, that to speak with a human is like a small vacation." It smiles, and you catch a glimpse of its massive incisors. Based on your knowledge of xenobiology you would guess that the Hai are herbivores, but you can't be certain.`
+			choice
+				`	"So I'm allowed to travel throughout your territory if I want?"`
+					goto north
+				`	"Can I ask more about the Unfettered now?"`
+					goto questions
+				`	"Thank you, but I have no more questions."`
+					decline
+			label questions
+			apply
+				set "Ask the Hai about the Unfettered: offered"
+			`	The Hai seems willing to explain more, what would you like to ask?`
+			choice
+				`	"Could you tell me about how the Unfettered came about?"`
+					goto raids
+				`	"They said the Drak altered the Hai to make them less aggressive. Is that true?"`
+					goto drak
+				`	"No, I want to know more about your society. Is it as peaceful as it appears?"`
+					goto peaceful
+				`	"That's all I wanted to know. Thanks."`
+					decline
+			label raids
+			`	He pauses. "Hundreds of years ago, our territory was raided by aliens from the east, who came in massive warships and plundered our worlds. In those days there were some Hai who were brave enough to risk their lives to defend us. The raids lasted for decades, then abruptly stopped.`
+			`	"Most of those who had fought the invaders were glad to return to ordinary life, but there were some for whom life now seemed bland and meaningless without the excitement and chaos of war. They started colonies in the systems to the north, and fought amongst themselves, and mismanaged the land, and grew in numbers. And now their worlds cannot feed them all, and they invade us and tell stories to put blame on us for their failures."`
+			choice
+				`	"Shouldn't they be treated with gratitude for defending you against the raiders?"`
+				`	"I have other questions."`
+					goto questions
```

Comment:
Don't loop back to questions, because otherwise we can very awkwardly ask the same question multiple times. Instead, simply follow the same structure as "Ask the Hai..."